### ISSUE TITLE
iframe w/in tinymce not accessible

### ISSUE THREAD

**Issue by eschiebel** [tinymce/tinymce#6486]
**What is the current behavior? Describe the bug**
Have content including an iframe, e.g. an embedded youtube video and add a title attribute, then edit in tinymce using a screenreader.
1. you either cannot use the keyboard to move to or past the iframe, the text cursor is stuck before the iframe, or the text cursor jumps over the iframe as if it wasn't there.
2. the span tinymce wraps around the iframe does not include the title attribute so the screen reader will not announce what this iframe is, say the title of the video

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](https://fiddle.tiny.cloud/) or similar.**
at https://fiddle.tiny.cloud/ using the source editor, insert

```
<h2>iframe example</h2>
this is before the iframe 
<iframe width="560" height="315" src="https://www.youtube.com/embed/1EKaflXTzHw" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen title="ginger baker drum solo video"></iframe>
this is after the iframe
```

and save. Then try to navigate the content in tinymce using the keyboard and screenreader

**What is the expected behavior?**
 - The iframe is announced as content by the screenreader
 - it's title attribute, if present is read
 - Using screenreader keyboard commands, I can navigate to and over the iframe.
 

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
MacOS Catalina 10.15.7, Safari 14.0.2 + VoiceOver, tinymce-cloud5-stable

**Comment by ltrouton** [comment]
Hi, thanks for posting.

I am unable to reproduce the keyboard navigation issues you have noted when using VoiceOver and MacOS BigSur. If you could include more details or a video showing how to replicate the keyboard navigation issue that would be helpful. It is worth noting that we need to rely largely on browser behaviour for the specified HTML. We may not be able to do anything about this, as if we made keyboard navigation work inside TinyMCE then it wouldn't be accessible once published.

With regards to the title of the iframe not being announced, we will have to investigate this further and let you know when we have any updates.

**Comment by MarcoZehe** [comment]
FWIW, I just tried this with Chromium Edge 89 and NVDA on Windows. The moment the cursor lands on the "character" that is the iframe, NVDA speaks "selected", meaning the whole iframe is now selected. I believe the iframe itself is static, meaning you cannot edit its contents. From the looks of the accessibility tree, that looks correct, and all the YouTube controls are accessible to me via NVDA's object navigator, or if I preview the content.
